### PR TITLE
Remove deprecated flash saving capabilities and tidy-up

### PR DESCRIPTION
**Pull request by 8ryn**
This removes the flash saving capabilities. Only merge this if we are certain this is what we want to do and we don't want to leave this functionality to make it easier for someone to re-implement it in future.

### CODE REVIEW

**Comment by 8ryn** [comment]
Will push further tidy up work to this branch so don't merge this quite yet!

**Comment by 8ryn** [comment]
Have now removed legacy files so happy to merge this after review.

**Review by ulrikpedersen — APPROVED** [review]
Agreed today with Robert and Armando to remove this unused, untested and possibly broken flash support.

**Pull request merged**
